findbugsTestCases/src/java/ghIssues/Issue0049.java

```diff
@@ -0,0 +1,33 @@
+
+import java.io.Serializable;
+
+import edu.umd.cs.findbugs.annotations.ExpectWarning;
+import edu.umd.cs.findbugs.annotations.NoWarning;
+
+public class Issue0049 {
+    public interface TestInterface {}
+
+    @NoWarning("RI_REDUNDANT_INTERFACES")
+    public static class SuperClass implements TestInterface {}
+
+    @ExpectWarning("RI_REDUNDANT_INTERFACES")
+    public static class SubClassWithDuplicate implements TestInterface {}
```

Comment:
I don't understand this test: why `public static class SuperClass implements TestInterface {}` should not produce the warning, but `public static class SubClassWithDuplicate implements TestInterface {}` should produce? What's the difference between these two?